docs/tutorial/remix-tips.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 7
+---
+
+# REMIX Tips
+
+## Generate and Deploy DApp UI with Quick DApp Plugin
+
+The Quick DApp plugin in REMIX simplifies the process of creating and deploying decentralized applications (DApps). This tool automates the generation of a user interface (UI) for your smart contracts and facilitates quick deployment to the Surge platform.
+
+1. Activate the Plugin
+
+   Open REMIX IDE and navigate to the **Plugin Manager** tab. Search for **Quick DApp** and click **Activate**.
+
+2. Select Your Contract
+
+   Choose the smart contract you want to deploy from the **Deploy & Run** tab. The plugin will automatically identify your contract and generate a corresponding UI.
+
+3. Configure the UI
+
+   You can customize the UI components based on the functions of your smart contract. The plugin allows you to specify which functions to expose to the user interface.
+
+4. Setup Surge
+
+   ```bash
+   # install surge
+   npm install --global surge
+
+   # create login
+   surge login
+   ```
+
+5. Deploy to Surge
+
+   Once your UI is configured, click the **Deploy** button. The plugin will handle the deployment process to Surge, providing you with a public URL to access your DApp.
+
+## Connect to Your Local Filesystem
+
+If you prefer working with contracts on your local filesystem, Remix can connect directly to it. Keep in mind that connecting to your local filesystem will switch your current workspace. If you want to keep working on contracts from your current workspace, make sure to download them first.
+
+![Download workspace](img/download.png)
+
+Note that switching to a local filesystem will cause all previously deployed contracts, both pinned and unpinned, to become invisible in the **Deploy & Run** tab. If you need to interact with already deployed contracts, make sure to note down their addresses beforehand so you can load them later using the **At Address** button.
+
+![Load contract](img/load_contract.png)
```

Comment:
I think we should keep this in the main tutorial. Telling people how to interact with pre-existing contracts is basic knowledge. Just as a separate main heading "## How to interact with already deployed contracts".

Reply:
done